sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/CustomRelationship.cs

```diff
@@ -2,15 +2,14 @@
 // Licensed under the MIT License.
 
 using System.Text.Json.Serialization;
-using Azure.DigitalTwins.Core.Serialization;
 
 namespace Azure.DigitalTwins.Core.Samples
 {
     /// <summary>
     /// Custom type for a sample illustrating how someone can create their own class to match a relationship type
     /// for serialization, instead of using <see cref="BasicRelationship"/>.
     /// </summary>
-    internal class CustomRelationship
+    internal class CustomRelationship : IDigitalTwinRelationship
     {
         [JsonPropertyName("$relationshipId")]
```

Comment:
why not use DigitalTwinsJsonPropertyNames.RelationshipId instead? (Same for the other properties below)